content/en/01-about-pixie/02-data-sources.md

```diff
@@ -34,9 +34,9 @@ Pixie automatically traces the following protocols:
 | Protocol      | Support             | Notes                          |
 | :------------ | :------------------ | :----------------------------- |
 | HTTP          | Supported           |                                |
-| HTTP2/gRPC    | Partially Supported | Currently only for Golang apps |
+| HTTP2/gRPC    | Partially Supported | Currently only for Golang apps with [debug symbols](/reference/admin/debug-symbols). |
```

Comment:
Should these two tables also mention dwarf symbols? This [README](https://github.com/pixie-io/pixie/blob/main/src/stirling/source_connectors/socket_tracer/protocols/http2/README.md) file mentions both are required.

Reply:
Can we call it debug information to be more precise?